tokio/src/process/mod.rs

```diff
@@ -994,13 +994,19 @@ impl Child {
     /// If the caller wishes to explicitly control when the child's stdin
     /// handle is closed, they may `.take()` it before calling `.wait()`:
     ///
-    /// ```no_run
+    /// ```
+    /// # #[cfg(not(unix))]fn main(){}
+    /// # #![cfg(unix)]
```

Comment:
You can't use the exclamation mark form in this case. Put it on the `main` function instead.
```suggestion
    /// # #![cfg(unix)]
```

Reply:
You're right! My bad!